docs/command_line_tool/phylum_extension_install.md

```diff
@@ -4,29 +4,32 @@ category: 6255e67693d5200013b1fa3e
 hidden: false
 ---
 
-Install an extension
+Install extension
 
 ```sh
-phylum extension install [OPTIONS] <PATH>
+Usage: phylum extension install [OPTIONS] <PATH>
 ```
 
+### Arguments
+
+<PATH>
+
 ### Options
-`--accept-permissions`
-&emsp; Automatically accept requested permissions
 
-`--overwrite`
-&emsp; Overwrite existing extension
+-y, --yes
+&emsp; Accept permissions and overwrite existing extensions (same as --overwrite --accept-permissions)
 
-`-y`, `--yes`
-&emsp; Accept permissions and overwrite existing (same as `--overwrite --accept-permissions`)
```

Comment:
The backticks got dropped here. It might be worth adding them back in at the command definition.